src/CloudService/Changelog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 1.0.0
+* GA release for module Az.CloudService.
```

Comment:
## Version 0.2.0
* Bump API version to stable 2021-03-01
* Supported OSFamily and OSVersion